sdk/confidentialledger/confidential-ledger-rest/src/confidentialLedger.ts

```diff
@@ -24,7 +24,11 @@ export default function ConfidentialLedger(
 
   const confidentialLedger = GeneratedConfidentialLedger(ledgerBaseUrl, creds, options);
 
-  confidentialLedger.pipeline.addPolicy(certificatePolicy(ledgerTlsCertificate, credentials));
+  if (isCertificateCredential(credentials)) {
+    confidentialLedger.pipeline.addPolicy(
+      getClientCertificatePolicy({ ...credentials, ca: ledgerTlsCertificate })
```

Comment:
I'm a little curious how we're going to generalize passing in the CA cert, since this currently feels a bit specific to confidential ledger - but maybe this is not common?